src/plugin/area/geojsonareaimportui.js

```diff
@@ -0,0 +1,90 @@
+goog.provide('plugin.area.GeoJSONAreaImportUI');
+
+goog.require('os.ui.im.FileImportUI');
+goog.require('os.ui.window');
+goog.require('os.ui.wiz.OptionsStep');
+goog.require('os.ui.wiz.step.TimeStep');
+goog.require('plugin.area.geojsonAreaImportDirective');
+goog.require('plugin.file.geojson.GeoJSONParserConfig');
+goog.require('plugin.file.geojson.geojsonImportDirective');
+
+
+
+/**
+ * @extends {os.ui.im.FileImportUI.<plugin.file.geojson.GeoJSONParserConfig>}
+ * @constructor
+ */
+plugin.area.GeoJSONAreaImportUI = function() {
+  plugin.area.GeoJSONAreaImportUI.base(this, 'constructor');
+
+  // file contents are only used in memory, not loaded from storage
+  this.requiresStorage = false;
+};
+goog.inherits(plugin.area.GeoJSONAreaImportUI, os.ui.im.FileImportUI);
+
+
+/**
+ * @inheritDoc
+ */
+plugin.area.GeoJSONAreaImportUI.prototype.getTitle = function() {
+  return 'Area Import - GeoJSON';
+};
+
+
+/**
+ * @inheritDoc
+ */
+plugin.area.GeoJSONAreaImportUI.prototype.launchUI = function(file, opt_config) {
+  plugin.area.GeoJSONAreaImportUI.base(this, 'launchUI', file, opt_config);
+  var steps = [
+    new os.ui.wiz.step.TimeStep(),
```

Comment:
The only step you should need is `os.query.ui.AreaOptionsStep`. Areas do not have a time component, and the current options step is for configuring layer options. The area options step allows the user to choose columns to populate the area name/description/tags and merge all areas in the file into one.